extensions/labgraph_protocol/README.md

```diff
@@ -0,0 +1,33 @@
+# LabGraph Protocol
```

Comment:
Do you want to add in more information explaining some key concepts used in this framework?
Examples could be: protocols, trials, conditions, stimuli, stimulus, results, identifier, etc. This could be useful for a wider audience who wants to use the framework.
Reference: https://github.com/facebookresearch/labgraph/blob/main/docs/concepts.md
You can also consider putting it in this folder or this folder for more visibility: https://github.com/facebookresearch/labgraph/tree/main/docs

Reply:
Agreed! Let me write up some docs in a separate PR.